### PR TITLE
Use APIs from eventing kafka broker in tests

### PR DESCRIPTION
**Pull request by mgencur**
Related to https://issues.redhat.com/browse/SRVKE-1367

## Proposed Changes

- references APIs from eventing-kafka-broker instead of eventing-kafka
- remove dependency on eventing-kafka
-

### CODE REVIEW

File: go.mod
Comment (creydr) [inline go.mod:1]: Can we remove the replace in line 185 too?

**Review by creydr — COMMENTED** [review]
Only one question. Otherwise looks good from my side

File: go.mod
Comment (mgencur) [reply go.mod:1]: Sure. Pushing.

**Review by creydr — APPROVED** [review]
/lgtm

**Pull request merged**